extra-types.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for intl-messageformat 1.3
```

Comment:
This is MIT licensed, I'm not sure about the correct way to redistribute this file.

Reply:
I actually do not know 😢

Reply:
I'll ask around and find an answer.

Reply:
@aomarks Did you receive any legal response regarding the license?

Reply:
My only concern is that there's no license data in this file, since I suppose usually implicit from the package.json and the top-level DefinitelyTyped readme when distributed through DT. Maybe just copy the generic DT license header in a comment block to the top of this file:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/LICENSE

Reply:
@aomarks Sounds good to me. PTAL 😄